cmd/execute.go

```diff
@@ -109,9 +112,18 @@ func executeTransformers() {
 	}
 
 	if len(ethStorageInitializers) > 0 {
-		storageHealthCheckMessage := []byte("storage watcher starting\n")
+		storageHealthCheckMessage := []byte("storage watcher for new diffs starting\n")
+		statusWriter := fs.NewStatusWriter(healthCheckFile, storageHealthCheckMessage)
+		sw := watcher.NewStorageWatcher(&db, newDiffBlockFromHeadOfChain, statusWriter, watcher.New)
+		sw.AddTransformers(ethStorageInitializers)
+		wg.Add(1)
+		go watchEthStorage(&sw, &wg)
+	}
+
+	if len(ethStorageInitializers) > 0 {
+		storageHealthCheckMessage := []byte("storage watcher for unrecognized diffs starting\n")
```

Comment:
This will probably need to happen in a separate PR against `vdb-mcd-transformers` (and get updated by tech ops in the deploy), but I imagine we'll want to update the health check to check for both messages

Reply:
I think that the transformers PR should take care of checking for the both message: https://github.com/makerdao/vdb-mcd-transformers/pull/277/files#diff-015ae56a7505cbe6e703b68709ad40e1. I'm not sure what we would need to have tech ops change though 🤔